README.md

```diff
@@ -37,7 +37,7 @@ The next mutation shows code from the `removeNode` method of a [linked list](htt
 }
 ```
 
-We know that the code originates from a remove method which means that the mutation introduces a leak by ignoring the removal of a reference. This can be [tested](https://github.com/zimmski/container/commit/142c3e16a249095b0d63f2b41055d17cf059f045) with [go-leaks](https://github.com/zimmski/go-leak).
+We know that the code originates from a remove method which means that the mutation introduces a leak by ignoring the removal of a reference. This can be [tested](https://github.com/avito-tech/container/commit/142c3e16a249095b0d63f2b41055d17cf059f045) with [go-leaks](https://github.com/avito-tech/go-leak).
```

Comment:
`https://github.com/avito-tech/container/commit/142c3e16a249095b0d63f2b41055d17cf059f045` is a broken link

Reply:
Same goes here `https://github.com/avito-tech/go-leak`

Reply:
Seems like reverting to zimmski is the best way